pkg/app/app_factory.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 VMware, Inc.
```

Comment:
2021?

Reply:
i was under impression we keep existing copyright notices when files get moved around.

Reply:
Yeah, not sure. Fine to keep unless we know otherwise.